app-android-xamarin/Moodis/Moodis/Feature/History/HistoryActivity.cs

```diff
@@ -39,7 +39,7 @@ private void InitView()
                 DatePickerFragment frag = DatePickerFragment.NewInstance(delegate (DateTime time)
                 {
                     dateInput.Text = time.ToLongDateString();
-                    (recyclerView.GetAdapter() as HistoryStatsAdapter).UpdateList(historyViewModel.FetchStats(SignInViewModel.currentUser.Id, time));
+                    (recyclerView.GetAdapter() as HistoryStatsAdapter).UpdateList(historyViewModel.FetchStats(GroupActivityModel.GetFriendIds(), time));
```

Comment:
Can the regular user still check his own activity ? And if not, he should.

Reply:
I could fix this up next sprint, when adding charts. I am thinking of implementing some kind of filtering here, so a user could quickly find required stats

Reply:
Yes. It's just a bad naming that I will fix, but it sorts through all the members that are in the same group, including the user itself